__tests__/app.test.js

```diff
@@ -92,15 +92,174 @@ describe("testing app /api/articles", () => {
           const articles = body;
           expect(articles).toHaveLength(12);
           expect(articles[0].created_at).toEqual("2020-11-03T09:12:00.000Z");
-          articles.forEach(article => {
-            expect.objectContaining(testArticleObj)
+          articles.forEach((article) => {
+            expect.objectContaining(testArticleObj);
           });
           // expect(articles).toEqual(
           //   expect.arrayContaining([expect.objectContaining(testArticleObj)])
           // );
         });
     });
   });
+  describe("testing GET on /api/articles with queries", () => {
+    describe("Sort queries defaulted to desc", () => {
+      test("should return an array of article objects, status 200 and be sorted by title", () => {
+        return request(app)
+          .get("/api/articles?sort_by=title")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].title).toEqual("Z");
+          });
+      });
+      test("should return an array of article objects, status 200 and be sorted by topic", () => {
+        return request(app)
+          .get("/api/articles?sort_by=topic")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].topic).toEqual("mitch");
```

Comment:
This test works for this case however it may break in future when there are more topics, is there a way we could make this more fluid? Perhaps jest toBeSortedBy would test the behaviour we want.